src/index.js

```diff
@@ -6,39 +7,18 @@ import { OverridableContext } from 'react-overridable';
 import { Router } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
 import { InvenioILSApp } from './lib';
+import { searchConfig } from './lib/config/searchConfig';
 
-// const CustomHome = ({ ...props }) => {
-//   return <>And this is custom home</>;
-// };
-//
-// const CustomCover = ({ size, url, isRestricted, asItem, linkTo }) => {
-//   const Cmp = asItem ? Item.Image : Image;
-//   const link = linkTo ? { as: Link, to: linkTo } : {};
-//   return (
-//     <Cmp
-//       centered
-//       disabled={isRestricted}
-//       {...link}
-//       onError={e => (e.target.style.display = 'none')}
-//       src={url}
-//       size="tiny"
-//     />
-//   );
-// };
-
-const overriddenCmps = {
-  // 'Home.layout': CustomHome,
-  // 'LiteratureCover.layout': CustomCover,
-};
+const overriddenCmps = {};
 
 const config = {
   invenioConfig: {
     authors: {
       maxDisplay: 42,
     },
   },
-  uiConfig: {},
-  searchConfig: {},
+  uiConfig: { ...uiConfig },
```

Comment:
this will be probably refactored, concerns the problems we talked about on the standup today

Reply:
What about `uiConfig: uiConfig`?

Reply:
does it make a difference now since it will be refactored anyway?